assets/sass/vendor/_mdc-button.scss

```diff
@@ -89,6 +89,11 @@
 		}
 	}
 
+	&--surfaces {
```

Comment:
Let's call this `inverse` to match the design token instead.

Reply:
Updated, thank you!